packages/react-magma-dom/src/components/Stepper/Stepper.tsx

```diff
@@ -0,0 +1,313 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+
+import { ThemeContext } from '../../theme/ThemeContext';
+import { ThemeInterface } from '../../theme/magma';
+import { useIsInverse } from '../../inverse';
+import { Step, StepProps, StepStatus } from './Step';
+import { transparentize } from 'polished';
+
+/**
+ * @children required
+ */
+export interface StepperProps extends React.HTMLAttributes<HTMLDivElement> {
+  testId?: string;
+  /*
+   * Custom number for specific breakpoint responsive styling
+   */
+  breakpoint?: number;
+  /*
+   * Determines the Stepper view based on a minimum breakpoint number.
+   */
+  breakpointStyle?: BreakPointStyle;
+  /*
+   * Sets a custom label for the Step count # of #.
+   */
+  stepDescriptionLabel?: string;
+  /*
+   * Sets a custom label for the completed Step.
+   */
+  completedStepDescription?: string;
+  /*
+   * Current step value
+   */
+  currentStep?: number;
+  /**
+   * @internal
+   */
+  hasBreakpoint?: boolean;
+  /*
+   * Inverse styling
+   */
+  isInverse?: boolean;
+  /*
+   * Hides the Step labels
+   */
+  hideLabels?: boolean;
+  /*
+   * Changes from Step labels on each Step to one label at a time.
+   */
+  summaryView?: boolean;
+  /**
+   * @internal
+   */
+  stepStatus?: StepStatus;
+  /**
+   * @internal
+   */
+  theme?: ThemeInterface;
+}
+
+export enum BreakPointStyle {
+  hideLabels = 'hideLabels',
+  summaryView = 'summaryView',
+}
+
+function StepSeparatorBackgroundColors(props: StepperProps) {
```

Comment:
Same comments in this file as on Step:
1. Lets use the enums instead of strings for comparisons
2. Functions should not be capitalized
3. Try destructuring prop names when it makes sense
4. Remove props being passed to styling constants when they are not used